### PR TITLE
Update Platforms.md

### PR DESCRIPTION
**Pull request by felipeplets**
### Checklist
- [x] I've run `bundle exec rspec` from the root directory to see all new and existing tests pass
- [x] I've followed the _fastlane_ code style and run `bundle exec rubocop -a` to ensure the code style is valid
- [x] I've read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I've updated the documentation if necessary.

### CODE REVIEW

File: fastlane/docs/Platforms.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Support for multiple platforms
 
-`fastlane` can handle multiple app platforms in one `Fastfile`. This is super useful for projects supporting iPhone, macOS and Android apps.
```
Comment (KrauseFx) [inline fastlane/docs/Platforms.md]: why the change from `and` to `or`?

Comment (taquitos) [inline fastlane/docs/Platforms.md]: could you add an oxford comma too 👍

Comment (felipeplets) [inline fastlane/docs/Platforms.md]: As I'm playing a lot with platforms in Fastlane and my main intention here was fix the iPhone/iOS issue.

While doing the change, I read the sentence and the "and" looked to me a bit odd because gave me the impression that all criterias should be met (iOS && macOS && Android), of course that is not a big issue and we can keep "and" if you think it's more suitable.

**Review by felipeplets — COMMENTED** [review]
I signed it!

**Comment by ohayon** [comment]
Hey @felipeplets - I am going to close this PR for now as it's just a bit stale, but please feel free to update and reopen it if you would like to make the changes mentioned in the comments. Thanks again for the contribution! We would love to see more! 🚀
